src/NServiceBus.Core/Persistence/InMemory/SagaPersister/InMemorySagaPersister.cs

```diff
@@ -171,6 +182,7 @@ static IContainSagaData DeepClone(IContainSagaData source)
             }
 
             public IContainSagaData SagaData;
+            public readonly string LockTokenKey;
```

Comment:
I've never understood our aversion to using `readonly`. What is the reason for not using it?